src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingTransactionOverPolicyByteLimitSpecification.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests.Wallet
+{
+    public partial class SendingTransactionOverPolicyByteLimit
```

Comment:
In Given When Then,  the `When` should be the last action you take before then asserting the side effects of the system using `Then`s.

This test blurs that a bit too much as that last action you can take (sending the transaction) is happening in the Then and the When/And also has an assert that should actually be part of the Givens.   It is fairly reasonable to have an assert at the end of a Given (like the asserting the tx is in the range 90-100kb as you did) but usually you only do this when writing the test to prove it to yourself and ensure the setup is correct, still I do think that is ok to keep as the next reader/writer of the code will like to see that too if not clear from the action of creating the tx that size. 

There perhaps something like the following would be a better logical breaking up of the Given When Then

```
Given two connected nodes
And node 1 builds a transaction just UNDER the 100kb limit
When sending the transaction
Then the transaction is successfully mined and propagated
```

```
Given two connected nodes
And node 1 builds a transaction OVER the 100kb limit
When sending the transaction
Then the transaction fails with oversize transaction error
```


You might be able to make use of sending

Reply:
Agreed.  Have followed the Given->And->When->Then pattern